### ISSUE TITLE
LDIF incorrectly generated when deleting DNs [v2.5.1]

### ISSUE THREAD

**Issue by QuantumFractal** [cannatag/ldap3#785]
Hello there,

I'm using LDAP3 to automate some cluster generation / teardown. I'm generating LDIFs in the following manner:
```python
ldif_gen = ldap3.Connection(server=None, client_strategy=ldap3.LDIF)

with ldif_gen:

       # Perform some ldif_gen.delete calls here

       result = ldif_gen.stream.getvalue()
       return result
```

I'm getting LDIFs that look like the following:
```
version: 1

dn: <DN 1>

changetype: delete

dn: <DN 2>

changetype: delete

dn: <DN 3>

changetype: delete
```
Which is currently breaking LDAPv3 spec. Is there something I'm missing or should I dig in an fix this.

As a quick asside, is there a way to disable line wrapping when writing out to an LDIF?

Thanks!
-Tom

**Comment by QuantumFractal** [comment]
Looking at the latest version of LDAP3, I think I've found the bug.
```
def delete_request_to_ldif(entry, all_base64, sort_order=None):
    lines = []
    if 'entry' in entry:
        lines.append(_convert_to_ldif('dn', entry['entry'], all_base64))
        lines.append(add_controls(entry['controls'], all_base64))
        lines.append('changetype: delete')
        if sort_order:
            lines = sort_ldif_lines(lines, sort_order)
    else:
        raise LDAPLDIFError('unable to convert to LDIF-CHANGE-DELETE - missing DN ')

    return lines
```

When using the LdapProvider strategy, this `delete_request_to_ldif` will add a blank line if there are no controls provided. With other operations there must be controls provided, however, with delete it can be empty.

**Comment by cannatag** [comment]
Can you test the code in dev? it should work now. 

Thanks.

**Comment by QuantumFractal** [comment]
Testing against `dev`

```
~/Repos/EXTERNAL/ldap3 dev*
venv ❯ pip freeze | grep ldap
ldap3==2.7

~/Repos/EXTERNAL/ldap3 dev*
venv ❯ python test_delete.py
version: 1

dn: CN=testLAB,CN=Users
changetype: delete

dn: CN=testLAB2,CN=Users
changetype: delete

dn: CN=testLAB3,CN=Users
changetype: delete
```

Looks good to me! Thanks!

**Comment by cannatag** [comment]
Fixed in ldap3 v2.7.
